boards/v2/qemu/qemu_xtensa/Kconfig.qemu_xtensa

```diff
@@ -0,0 +1,10 @@
+# XTENSA board configuration
+
+# Copyright (c) 2017, 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_QEMU_XTENSA
+	bool
+
+config BOARD_QEMU_XTENSA_DC233C_MMU
+	bool
```

Comment:
Not needed, this is handled here: 
https://github.com/zephyrproject-rtos/zephyr/blob/051c62a76a7d54f04893f6d73f77c1f946493d04/boards/Kconfig.v2#L8
https://github.com/zephyrproject-rtos/zephyr/blob/051c62a76a7d54f04893f6d73f77c1f946493d04/boards/Kconfig.v2#L13

```suggestion
```

Reply:
Fixed